addons/event_booth_sale/models/sale_order_line.py

```diff
@@ -74,14 +77,6 @@ def _onchange_event_id_booth(self):
         if self.event_booth_pending_ids and (not self.event_id or self.event_id != self.event_booth_pending_ids.event_id):
             self.event_booth_pending_ids = None
 
-    @api.depends('event_booth_pending_ids')
```

Comment:
It occurred to me (in a dream) that this is a computed field. So I'm guessing that's actually the reason it keeps on triggering the compute as it "changes" every time it needs to be recomputed, which is every rpc call.

Changing this dependency to `event_booth_registration_ids.event_booth_id` instead, which is exactly what it reduces to, seems to fix the issue on my end. I think it's better to do it this way if it works for you, as it ensures we keep calling this when relevant in case people were relying on that for a custom addon. And we avoid setting irrelevant fields in the inverse (name isn't used to compute pending ids so conceptually the inverse shouldn't touch it either)

Reply:
Fix seems to work and is much cleaner.